src/api/nlopt-in.hpp

```diff
@@ -598,6 +603,10 @@ namespace nlopt {
       get_initial_step(x, v);
       return v;
     }
+
+    // exceptions in opt::optimize:
+    bool get_exceptions_enabled() const { return exceptions_enabled; }
+    void set_exceptions_enabled(bool enable) { exceptions_enabled = enable; }
```

Comment:
these must be documented

Reply:
Followup commit e94cd1f3d1d106cfc603713b72bed16a68da862c adds the missing documentation for the affected languages (C++, Guile, Python). (Note that the Java documentation will need basically the same changes as the Python one, but the Java documentation is not in `master` yet and I do not want the pull requests to depend on each other.)

Reply:
I have the Java documentation in my fork as commit d29bae391e6d92d632b33b01876e409d354bca12, I can cherry-pick it somewhere when one or both of the pull requests are merged.